File: web/back/python-api/src/status/status_service.py
"""
Datei: status_service.py

Zweck:
Diese Datei baut die Status-Antwort für das Frontend.
"""

from src.database.database_service import check_database_connection


def get_status() -> dict:
    database = check_database_connection()

    return {
        "backend": "Python FastAPI",
        "greeting": "Hallo Gast",
        "status": "online",
        "database": database,
    }

File: web/back/java-api/src/main/java/com/example/javaapi/status/StatusService.java
package com.example.javaapi.status;

import com.example.javaapi.database.DatabaseService;
import org.springframework.stereotype.Service;

import java.util.Map;

/**
 * Datei: StatusService.java
 *
 * Zweck:
 * Diese Klasse baut die Status-Antwort für das Frontend.
 */
@Service
public class StatusService {

    private final DatabaseService databaseService;

    public StatusService(DatabaseService databaseService) {
        this.databaseService = databaseService;
    }

    public Map<String, Object> getStatus() {
        return Map.of(
                "backend", "Java Spring Boot",
                "greeting", "Hallo Gast",
                "status", "online",
                "database", databaseService.checkConnection()
        );
    }
}